### PR TITLE
Mes-6984: refactor ion buttons

### PR DESCRIPTION
**Pull request by andrewsetterfield**
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
Screenshot 2021-07-02 at 07 57 19
Screenshot 2021-07-02 at 07 58 11
Screenshot 2021-07-02 at 07 58 28
Screenshot 2021-07-02 at 08 01 36

### CODE REVIEW

File: src/app/pages/dashboard/components/test-results-search-card/test-results-search-card.scss [hunk 1]
```diff
@@ -54,7 +48,7 @@ button[clear] {
     display: flex;
     align-items: center;
     height: 240px;
-    padding: 24px;
+    //padding: 24px;
```
Comment (Cullima) [inline src/app/pages/dashboard/components/test-results-search-card/test-results-search-card.scss]: can you remove if not needed?

File: src/app/pages/dashboard/dashboard.page.ts [hunk 2]
```diff
@@ -83,4 +83,7 @@ export class DashboardPage extends BasePageComponent {
 
   showDelegatedExaminerRekey = (): boolean =>
     this.appConfigProvider.getAppConfig().role === ExaminerRole.DLG;
+
+  testResultColSize = (): number =>
```
Comment (Cullima) [inline src/app/pages/dashboard/dashboard.page.ts:87]: have you checked this in landscape orientation?

**Review by Cullima — CHANGES_REQUESTED** [review]

File: src/app/pages/dashboard/components/test-results-search-card/test-results-search-card.scss (hunk 1 above)
Comment (andrewsetterfield) [reply src/app/pages/dashboard/components/test-results-search-card/test-results-search-card.scss]: removed

File: src/app/pages/dashboard/dashboard.page.ts (hunk 2 above)
Comment (andrewsetterfield) [reply src/app/pages/dashboard/dashboard.page.ts:87]: works in landscape

**Review by Cullima — APPROVED** [review]

**Review by gittins7 — APPROVED** [review]

**Pull request merged**